CHANGELOG.md

```diff
@@ -27,7 +35,15 @@ FEATURES
 * [server] Default config now creates a profiler at port 6060, and increase p2p send/recv rates
 * [tests] Add WaitForNextNBlocksTM helper method
 
-FIXES
+FIXES 
+* [gaia] Added self delegation for validators in the genesis creation
+* [cli] fixed cli-bash tests
+* [ci] added cli-bash tests
+* [basecoin] updated basecoin for stake and slashing
+* [docs] fixed references to old cli commands
```

Comment:
Did that happen in this PR? Some of these changelog entries look unrelated

Reply:
yeah not sure how that got in there, removed the duplicate lines